integration-test/src/main/java/com/sequenceiq/it/cloudbreak/CloudbreakTestSuiteInitializer.java

```diff
@@ -47,8 +45,8 @@
     @Value("${integrationtest.testsuite.cleanUpOnFailure}")
     private boolean cleanUpOnFailure;
 
-    @Value("${integrationtest.defaultBlueprintName}")
-    private String defaultBlueprintName;
+    @Value("${integrationtest.defaultClusterDefinitionName}")
```

Comment:
Are these tests triggered now by jenkins? If yes, are the jenkins jobs updated?

Reply:
I did not find `defaultBlueprintName` reference at hortonworks/cloudbreak-ansible-playbooks repository. So I suppose we do not need additional update.

Reply:
alrighty